src/Components/Pages/Pricing.tsx

```diff
@@ -9,44 +9,45 @@ import {
   ListGroupItem,
 } from "react-bootstrap";
 
+interface PricingState {
+  tiers: any[];
```

Comment:
Should be straightforward to define a type for the tiers

type tier {
     name: string,
     price: string, 
     perks: string[],
}

and then you can do
interface PricingState {
     tiers: tier[],
}

btw, should be a comma after the items inside an interface not a ;

Reply:
I realized it can also be a `;` and thats what prettier defaults to so that should be fine too